oreilly-scenarios/cockroach-region-failure/exploring-friendpay.md

```diff
@@ -0,0 +1,91 @@
+# Exploring FriendPay via CockroachDB
+
+Now that we have grasped some of the ways we can simulate failure in CockroachDB, as well as the value of geopartitioning in CockroachDB, let's combine that knowledge and see how our application responds to load.
```

Comment:
"we have grasped" -> "you have seen"

Reply:
In general, try to keep it to second-person rather than first-person plural.
we -> you
our -> your
"let us" -> ""
"let's" -> ""
etc.